renv.lock

```diff
@@ -1,6 +1,6 @@
 {
   "R": {
-    "Version": "4.2.2",
+    "Version": "4.3.2",
```

Comment:
Updating R in the lockfile was an accidental change; I can undo it if you expect it'll cause problems, but I figured we might as well update.